### DISCUSSION TITLE
Publishing to IIS 8 in Windows 2012

### DISCUSSION THREAD

**Discussion by bkotar1** [Dotnet-Boxed/Templates#183] [Q&A]
﻿**Which project template is the issue about?**
We used the MVC 6 template and trying to put the website in IIS. I am getting errors and unable to run it in IIS server. I am getting access denied error or http 500 error. Do you have any documentation to install it in IIS.
...

**Which version of the project template is the issue about?**

...

**Which version of Visual Studio or the dotnet SDK were you using?**
We are using the .net 461 with net core 1.1
...

**Comment by RehanSaeed** [comment]
There is some help in the ReadMe.html file. Consider turning on some logging and taking a look at the exception messages. I can't help any more than that with the limited amount of information in this issue.
